packages/server/controllers/posts.js

```diff
@@ -0,0 +1,64 @@
+const Post = require('../models/Post')
+
+module.exports = {
+    
+    createPost: async (req, res) => {
+        try {
+            const post = await Post.create({
+                title: req.body.title,
+                body: req.body.body,
+                isDraft: req.body.isDraft,
+            })
+
+            res.json({message: 'Post Created!', post})
+
+        } catch (error) {
+            console.log(error)
+        }
+    },
+    editPost: async (req, res) => {
+
+        const {title, body, isDraft} = req.body
+        const id = req.params.id
+
+        try {
+            if(title){
+                // try catch that updates title
+                try {
+
+                    await Post.findOneAndUpdate({_id: id},{title: title},{new: true})
+
+                } catch (error) {
+                    console.log(error)
+                }
+            }
```

Comment:
This is a bit repetative, see if we can dry it up by doing something like:
```js
const changeObject = {};
if (title) {
  changeObject.title = title
}
```

This is the most nieve way might want to see if you can optimize it even more.